stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageList.kt

```diff
@@ -261,6 +264,27 @@ public fun Messages(
     }
 }
 
+/**
+ * The default empty placeholder that is displayed when there are no messages in the channel.
+ *
+ * @param modifier Modifier for styling.
+ */
+@Composable
+internal fun DefaultMessageListEmptyView(modifier: Modifier) {
+    Column(
```

Comment:
I don't think we need a column given that we're showing just one item - maybe a `Box` here makes more sense/is a bit less confusing. As with a column you'd expect more elements.